### ISSUE TITLE
Undefined variable $vars

### ISSUE THREAD

**Issue by Jensderond** [studioespresso/craft3-dumper#13]
Hello, since craft 4 we aren't able to use the basic `{{ dump(item) }}` anymore because of the error:
`Undefined variable $vars` [on line 108 of DumperExtension.php](https://github.com/studioespresso/craft3-dumper/blob/develop/src/web/twig/DumperExtension.php#L108)

We are using:

- Craft 4.0.2
- PHP 8.1.6
- Dumper 3.0.0
- Twig v3.3.10

**Comment by jerome2710** [comment]
@janhenckens could this be fixed, please? `\studioespresso\craftdumper\web\twig\DumperExtension::dump()` seems to be missing its arguments. Based on the Craft 3 version I suspect it should be:

```
public function dump(\craft\web\twig\Environment $env, $context, ...$vars)
```

**Comment by janhenckens** [comment]
Fixed in 3.0.1.
